### PR TITLE
Add tests AddClassCommandParserTest and AttendanceTest

### PR DESCRIPTION
**Pull request by zixin448**

### CODE REVIEW

File: src/main/java/seedu/address/model/tutorial/Tutorial.java [hunk 1]
```diff
@@ -93,7 +93,7 @@ public Tutorial(TutorialName name, Venue v, Day d, Time t, int weeks, FilteredLi
      */
     public Tutorial(TutorialName name, Venue v, Day d, Time t, int weeks, AttendanceList attendance,
                     AssessmentResultsList results) {
-        requireAllNonNull(name, d, t, results);
+        requireAllNonNull(name, d, t, weeks, results);
```
Comment (honganhcs) [inline src/main/java/seedu/address/model/tutorial/Tutorial.java:96]: thanks for adding checks for the other parameteres! should we check for attendance too?

Comment (zixin448) [reply src/main/java/seedu/address/model/tutorial/Tutorial.java:96]: Ahh right will add it!

File: src/test/java/seedu/address/logic/commands/CommandTestUtil.java [hunk 2]
```diff
@@ -42,6 +47,16 @@
     public static final String VALID_ADDRESS_BOB = "Block 123, Bobby Street 3";
     public static final String VALID_TAG_HUSBAND = "husband";
     public static final String VALID_TAG_FRIEND = "friend";
+    public static final String VALID_TUTORIALNAME_T01 = "T01";
```
Comment (honganhcs) [inline src/test/java/seedu/address/logic/commands/CommandTestUtil.java:50]: As there are AssessmentTestUtil, AttendanceTestUtil classes, should we create a separate TutorialTestUtil for these variables to be consistent?

**Review by honganhcs — COMMENTED** [review]
there are some things you can consider changing, otherwise looks good :)

**Review by honganhcs — APPROVED** [review]

**Pull request merged**